app/lib/config/config.go

```diff
@@ -288,3 +289,27 @@ func InputYN(def bool, args ...interface{}) bool {
 	}
 	return strings.ToLower(string(Input(defStr, args...)[0])) == "y"
 }
+
+// Copy makes a copy of a local file with the same file-mode-bits set.
```

Comment:
and `If dst is a directory, it will copy the src file into dst`, or `it mimics the posix "cp" command` or something else saying the same thing :D